ttfb.sql

```diff
@@ -16,7 +16,7 @@ SELECT
    WHEN platform = 'squarespace' THEN 'Squarespace'
    WHEN platform = 'x-wix-request-id' THEN 'Wix'
    WHEN platform = 'x-shopify-stage' THEN 'Shopify'
-   WHEN platform = 'x-now-id' THEN 'ZEIT Now'
+   WHEN platform = 'x-now-id' AND platform <> 'x-now-instance' THEN 'ZEIT Now'
```

Comment:
Platform is only one value at a time, so this condition would always be true when `platform = 'x-now-id'`.

Reply:
Agreed. I added a second column called `secondary`.